.rubocop.yml

```diff
@@ -0,0 +1,15 @@
+---
+# Managed by modulesync - DO NOT EDIT
+# https://voxpupuli.org/docs/updating-files-managed-with-modulesync/
+
+inherit_gem:
+  voxpupuli-test: rubocop.yml
+
```

Comment:
@ekohl ehat do you think about this? at the moment we only use it in modules. other gems have their own rubocop config which is isually way shorter.

Reply:
I do think our Rubocop in vp-test is intended for Puppet modules, not real Ruby code. It would be better to come up with a shared config for real gems. It also would prevent this from pulling in a LOT of unrelated code.

Reply:
I'm fairly sure that as-is is a bad idea.  My goal with this PR was to simply get rubocop working and merged before the style changes bit-rot.  Then additional PRs can be made as cop configuration is agreed upon.  Configuring rubocop is like ordering a pizza... I was concerned that starting from scratch was going to take a long time to build consensus. I have two suggestions to resolve the circular dependency and I honestly don't have much of a preference.

A) move `rubocop.yml` out of `voxpupuli-test:` into a new gem
B) copy the current `voxpupuli-test` `rubocop.yml` into this module with minimal modifications

Are either of those acceptable solutions?

Reply:
@ekohl If I copy the `rubcop.yml` from `voxpupuli-test`, would this PR be ready for merge?